src/components/Progress.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { CircularProgress } from "@mui/material";
+import customColors from "../customColors";
+
+const Progress = (props) => {
```

Comment:
Also, you could receive and create variable using destructuring technique:

```
const Progress = ({ userProgress, quizId }) => {
```

Reply:
Great point @in43sh ! Destructuring the component params makes the component arguments more clear to me. If the props aren't descructured then I need to read through the entire component to understand which attributes of the props object are used in the component. If the props are desctructured in the definition of the component I only need to read the first line of the component to know which attributes are used.